### DISCUSSION TITLE
连接线 与 链接桩 有间隙

### DISCUSSION THREAD

**Discussion by JoeABCDEF** [antvis/X6#3251] [Q&A]
<img width="68" alt="image" src="https://user-images.githubusercontent.com/39638510/219543412-add5fe58-4e74-4973-bc87-509518e5839c.png">
<img width="151" alt="image" src="https://user-images.githubusercontent.com/39638510/219543429-8d83a2f5-eeec-4ef8-8010-8cc3a3cb2d1c.png">

希望在不显示箭头的情况下 与链接桩重合至少最外边 红色那个, 请问如何解决

**Comment by NewByVector** [comment]
默认不应该有这个间隙的，可以提供一个可以复现的 demo 看下。
